src/Parallel/Actions/SetupDataBox.hpp

```diff
@@ -88,19 +98,35 @@ struct SetupDataBox {
                     const ArrayIndex& /*array_index*/,
```

Comment:
Please update the documentation.  Make clear that the PDAL and ActionList do not depend upon the DataBox type (This is important to avoid a cyclic compilation dependency)